src/sasjs-help/index.js

```diff
@@ -150,6 +150,10 @@ export async function printHelpText() {
     )} - Lets the user run a given SAS file against a specified target.
     The target can exist either in the local project configuration or in the global .sasjsrc file.
 
+    * ${chalk.greenBright(
+      'request <sasJob> -d <path/to/datafile> -c <path/to/configfile> -t <targetName>`.'
+    )} - Lets the user run a SAS job against a specified target.
+    The target can exist either in the local project configuration or in the global .sasjsrc file.
```

Comment:
`<sasJob>` is confusing - is it a UUID, a job name?  Actually it is the full path and name.

Presumably `-d`, `-c` and `-t` are all optional?